script.js

```diff
@@ -0,0 +1,85 @@
+const form=document.getElementById("message-form");
+const messageContainer= document.getElementById("message-containre")
+const latestMessageContainer= document.getElementById("latest-message-containre")
+const nameId = document.getElementById("name-id");
+const messageId = document.getElementById("message-id");
+let formFrom = "";
+let formText = "";
+
+const fetchMessages = async() => {
+    const response = await fetch("http://localhost:3000/messages");
```

Comment:
use a const to store the API server host and port info. Avoid manually repeating the host info